pages/03-core/no-data-states.page.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Highcharts from "highcharts";
+import { omit } from "lodash";
+
+import Box from "@cloudscape-design/components/box";
+
+import CoreChart from "../../lib/components/internal-do-not-use/core-chart";
+import { PageSettings, PageSettingsForm, useChartSettings } from "../common/page-settings";
+import { FramedDemo, Page, PageSection } from "../common/templates";
+
+interface ThisPageSettings extends PageSettings {
+  showInLegend?: boolean;
+}
+
+export default function () {
+  const { chartProps } = useChartSettings<ThisPageSettings>();
+
+  const noDataContent = (
+    <Box textAlign="center" color="inherit">
+      <b>No data available</b>
+      <Box variant="p" color="inherit">
+        There is no available data to display
+      </Box>
+    </Box>
+  );
+
+  return (
+    <Page
+      title="Core chart: no data states"
+      subtitle="The page demonstrates all possible no-data states of the core chart."
+      settings={<PageSettingsForm selectedSettings={["height", "showLegend"]} />}
+    >
+      <PageSection title="Empty state: all" subtitle="No series provided">
+        <FramedDemo>
+          <CoreChart
+            {...omit(chartProps.cartesian, "ref")}
+            highcharts={Highcharts}
+            options={{ series: [] }}
+            noData={{ statusType: "finished", empty: noDataContent }}
+          />
+        </FramedDemo>
+      </PageSection>
+
+      <PageSection title="Empty state: all (side legend)" subtitle="No series provided">
+        <FramedDemo>
+          <CoreChart
+            {...omit(chartProps.cartesian, "ref")}
+            highcharts={Highcharts}
+            options={{ series: [] }}
+            legend={{ ...chartProps.cartesian.legend, position: "side" }}
+            noData={{ statusType: "finished", empty: noDataContent }}
+          />
+        </FramedDemo>
+      </PageSection>
+
+      <PageSection title="Empty state: data" subtitle="No data provided">
```

Comment:
The "no series" and "no data" empty states are not much different, unless you include axes configuration. I suggest to either only keep one, or make them distinct.

Reply:
So we can have one example w/o axes (so that HC does not know whether it is cartesian series at all or something else). Another example can use an empty line series and axes - then it will show axes and ticks, overlayed by the empty message.